setup.py

```diff
@@ -33,9 +33,10 @@ def load_requirements(*requirements_paths):
             )
     return list(requirements)
 
+
```

Comment:
nit: remove blank line

Reply:
That was actually intentional: Pylint expects an extra line before the `setup` function so this removes a warning message.